Settings.ui

```diff
@@ -249,6 +249,110 @@
                 </child>
               </object>
             </child>
+            <child>
+              <object class="GtkListBoxRow" id="general_row_4">
+                <property name="activatable">False</property>
+                <property name="focusable">False</property>
+                <property name="tooltip_text" translatable="yes">Comma separated values representing the "useful window widths" that will be cycled through. Values can be either ratio or pixel values. Mixed ratio and pixel values are not supported.</property>
```

Comment:
> ratio or pixel values

For consistency with #453, I'd suggest only supporting `___px` and `__%`, not plain numbers.

Reply:
yeah, I agree, especially if #453 switching to 'px' or '%'.

Cheers.